GameData/ETT/EngTechTree.cfg

```diff
@@ -912,6 +906,8 @@ TechTree
 			part = a_6m_cargo
 			part = a_cockpit_adaptor
 			part = B9_Cockpit_MK5
+			part = Airlock

```

Comment:
Is this a command module or a docking mechanism?
If the later, Docking ports might be a better node for it.:)